src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashPostBuildComment.java

```diff
@@ -70,7 +70,8 @@ public StashPostBuildComment newInstance(StaplerRequest req, JSONObject formData
     }
 
     @Override
-    public boolean isApplicable(Class<? extends AbstractProject> jobType) {
+    public boolean isApplicable(
+        @SuppressWarnings("rawtypes") Class<? extends AbstractProject> jobType) {
```

Comment:
same thing here, if this comes from super definition, why do we need to suppress it?

Reply:
Yes, it comes from the superclass:
https://javadoc.jenkins-ci.org/hudson/tasks/BuildStepDescriptor.html#isApplicable-java.lang.Class-

Reply:
Eclipse doesn't actually suppress this warning, but I can live with it. It's likely an Eclipse bug.